Example/FirstLaunchViewController.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+import MiniApp
+
+class FirstLaunchViewController: UIViewController {
+
+    @IBOutlet weak var imageViewArrow: UIImageView!
+    @IBOutlet weak var labelHint: UILabel!
+    @IBOutlet weak var labelMiniApp: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        guard Config.userDefaults?.string(forKey: Config.Key.applicationIdentifier.rawValue) != nil, Config.userDefaults?.string(forKey: Config.Key.subscriptionKey.rawValue) != nil else {
+            UIView.animate(withDuration: 1.5) {
+                self.imageViewArrow.alpha = 1.0
+                self.labelHint.alpha = 1.0
+            }
+            UIView.animate(withDuration: 0.5) {
+                self.labelMiniApp.alpha = 0.0
+            }
+            return
+        }
```

Comment:
Maybe we can move this to a separate function?

Reply:
sure

Reply:
done